awscrt/io.py

```diff
@@ -250,6 +250,14 @@ class TlsVersion(IntEnum):
     DEFAULT = 128  #:
 
 
+class TlsCipherPref(Enum):
+    DEFAULT = (0)
+    PQ_TLSv1_0_2021_05 = (6)
+
+    def __init__(self, tlsCipherEnumVal):
+        self.isSupported = _awscrt.is_tls_cipher_supported(tlsCipherEnumVal)
```

Comment:
I'd do this as a member function, instead of an attribute. It's transparent, the action in python maps to the action in C. Also, it lets you just make this a normal IntEnum, you don't have to mess with the `__init__` function

also, our python API uses snake_case, not camelCase

also, need to include a docstring or it won't show up in the API docs

```
    def is_supported(self):
        """Return whether this Cipher Preference is available in the underlying TLS implementation"""
        return _awscrt.is_tls_cipher_supported(self.value)

```